org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ClassActionImpl.java

```diff
@@ -1132,7 +1132,16 @@ private Object transformElementValue(Object inputValue) {
 			return outputElementValues;
 
 		} else {
+                    if (inputValue instanceof String) {
```

Comment:
Could we not follow the normal `else if (...) {` flow like the previous blocks?

Reply:
We should also move this above the `else if (inputValue instanceof Object[])` so that all the scalar types are done before the array.

Reply:
Ok, done.